doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,255 @@
+| File                        | File format                          | Programming scenario                                               |
++=============================+======================================+====================================================================+
+| :file:`merged.hex`          | Full image, HEX format               | Using an external debug probe and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_signed.hex`      | MCUboot compatible image, HEX format | Using the built-in bootloader and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_update.bin`      | MCUboot compatible image, HEX format | Using the built-in bootloader and the mcumgr command line tool     |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`dfu_application.zip` | MCUboot compatible image, binary     | Using nRF Programmer for Android and iOS or nRF Connect Programmer |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:53.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   If you do not have an external debug probe available to program the Thingy:53, you can directly program the device over USB using the built-in MCUboot bootloader and nRF Connect Programmer.
+   In this scenario, use the :file:`dfu_application.zip` file.
+
+.. _thingy53_build_pgm_segger:
+
+Building and programming using SEGGER Embedded Studio
+*****************************************************
+
+.. include:: gs_programming.rst
+   :start-after: build_SES_projimport_open_start
+   :end-before: build_SES_projimport_open_end
+
+..
+
+   .. figure:: images/ses_thingy_configuration.png
+      :alt: Opening the Asset tracker application
+
+      Opening the Asset tracker application for the thingy91_nrf9160_ns build target
+
+   .. note::
+
+      The *Board Directory* folder can be found in the following location: ``ncs/nrf/boards/arm``.
+
+4. Click :guilabel:`OK` to import the project into SES.
+   You can now work with the project in IDE.
+#. To build the sample or application:
+
+   a. Select your project in the Project Explorer.
+   #. From the menu, select :guilabel:`Build` > :guilabel:`Build Solution`.
+      This builds the project.
+
+   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the :file:`zephyr` subfolder in the build directery.
```

Comment:
```suggestion
   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the :file:`zephyr` subfolder in the build directory.
```
Alternatively: The output of the build with the merged HEX file containing... is located...